### DISCUSSION TITLE
How to use it?

### DISCUSSION THREAD

**Discussion by Shivanshu-S-AI** [Tyrrrz/YoutubeDownloader#371] [General]
my real question actually is, how to use it? 

I really don't know. 

There is neither .exe file, nor .py file. 

Please help!

**Comment by Tyrrrz** [comment]
https://github.com/Tyrrrz/YoutubeDownloader/releases

**Comment by ghost** [comment]
Hello sir🤗, Slava Ukraini. 
Sorry for causing you trouble. 
Sir, how can I use it on Windows 10? 
I extracted the zip file but I don't know which file should I have to click to start program. 
Sorry for my bad English. Hope you are doing well.

**Reply by Tyrrrz** [reply]
Make sure you download `YoutubeDownloader.zip` from [here](https://github.com/Tyrrrz/YoutubeDownloader/releases/latest) and then just run `YoutubeDownloader.exe`

**Comment by ghost** [comment]
Ahh ... My apologies. I wrongly downloaded the master branch codes zip file not the actual compiled file (release file). Thank you. Have a nice day, sir.
